python/python/bystro/vcf_utils/simulate_random_vcf.py

```diff
@@ -0,0 +1,65 @@
+"""VCF index and data simulator for testing analysis modules."""
+
+import random
+from io import StringIO
+
+import pandas as pd
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
+
+
+def generate_random_vcf_index() -> tuple[int, int, str, str]:
+    """Generate autosomal chr vcf variant IDs to use for testing."""
+    random_chr = random.randint(1, 22 + 1)
+    random_pos = random.randint(1, 1000000)
+    random_ref = random.choice(["A", "T", "C", "G"])
+    random_alt = random.choice([letter for letter in ["A", "T", "C", "G"] if letter != random_ref])
+    return random_chr, random_pos, random_ref, random_alt
+
+
+def generate_simulated_vcf(num_samples: int, num_vars: int) -> tuple[list[str], list[str]]:
+    """VCF simulator for testing analysis modules."""
+    # This is a first pass - could include more sophisticated choices for possible values in future
+    sample_ids = [f"SampleID{i+1}" for i in range(num_samples)]
+    header = HEADER_COLS + sample_ids
+    vcf_data = []
+    simulated_indices = []
+    vcf_data.append("\t".join(header))
+    for variant in range(num_vars):
+        # Use index simulator to generate random chrom,pos,ref,alt
+        random_chr, random_pos, random_ref, random_alt = generate_random_vcf_index()
```

Comment:
if we just name these vars like we want them here, we don't have to reassign them in L32-36